.travis.yml

```diff
@@ -13,3 +12,10 @@ before_install:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
   - sleep 3 # give xvfb some time to start
+
+script:
+  - npm run test:$TEST_MODE
```

Comment:
Because `npm test` hard codes the TEST_MODE (so that a plain npm test actually works / does something).